DigitalLearningSolutions.Web/Controllers/SupervisorController/Supervisor.cs

```diff
@@ -828,7 +828,7 @@ public IActionResult EnrolDelegateConfirm(int delegateUserId, int supervisorDele
                     GetCentreId()
                 );
             }
-            TempData.Clear();
+

```

Comment:
Not sure we should be retaining the tempdata at the end of this flow in order to solve this problem. I think we should be checking whether it exists on the in the controller method for the page we are going back to and throw a 410 expired error if so. We use an action filter elsewhere to achieve this.